packages/create-sitecore-jss/src/templates/nextjs-sxa/src/lib/page-props-factory/plugins/error-pages.ts

```diff
@@ -0,0 +1,44 @@
+import { GetServerSidePropsContext, GetStaticPropsContext } from 'next';
+import { SitecorePageProps } from 'lib/page-props';
+import { Plugin } from 'lib/page-props-factory';
+import { GraphQLErrorPagesService } from '@sitecore-jss/sitecore-jss-nextjs';
+import { isServerSidePropsContext } from '..';
+import config from 'temp/config';
+
+class ErrorPagesPlugin implements Plugin {
+  order = 2;
```

Comment:
Let's bump this to 3 to be safe (personalize is 2 as well)
```suggestion
  order = 3;
```